tests/mock_network/utils.rs

```diff
@@ -766,8 +775,11 @@ pub fn verify_section_invariants_between_nodes(nodes: &[TestNode]) {
     };
     let mut sections: BTreeMap<Prefix<XorName>, NodeSectionInfo> = BTreeMap::new();
 
-    for node in nodes.iter() {
-        let our_prefix = unwrap!(node.inner.our_prefix());
+    for node in nodes.iter().filter(|node| node.inner.is_elder()) {
+        let our_prefix = match node.inner.our_prefix() {
+            Some(pfx) => pfx,
+            None => continue,
```

Comment:
How can this happen? We are iterating only elders.

Reply:
Ah, right. I changed this first and only modified the iteration to be over elders later, and forgot about this part.